easybuild/tools/module_generator.py

```diff
@@ -52,24 +55,28 @@ class ModuleGenerator(object):
     """
     Class for generating module files.
     """
+    SYNTAX = None
 
     # chars we want to escape in the generated modulefiles
     CHARS_TO_ESCAPE = ["$"]
```

Comment:
@pforai: this is specific to Tcl module files? no need to escape `$` in Lua module files?

if so, move this to `ModuleGeneratorTcl` and set it to `None` here